utils/build_swift/driver_arguments.py

```diff
@@ -262,6 +262,12 @@ def create_argument_parser():
              "library for. %(default)s by default.",
         action=arguments.action.concat, type=arguments.type.shell_split,
         default=None)
+    targets_group.add_argument(
+        "--extra-stdlib-deployment-targets",
+        help="list of additional targets to cross-compile the Swift standard "
+             "library for in addition to default",
+        action=arguments.action.concat, type=arguments.type.shell_split,
+        default=None)
```

Comment:
If this is meant as a list of deployment targets, why not make the default an empty list. I plan to change the `--stdlib-deployment-targets` option to do the same.

Reply:
If possible would you mind also following the style changes from #12954, namely putting the help string last and action first in the list of kwargs.

Reply:
Yeah empty list would work too.